app/src/App.js

```diff
@@ -3,36 +3,52 @@ import "./App.css";
 import * as React from "react";
 
 import Account from "./Account.js";
+import gcal from "./api/ApiCalendar";
 import Planner from "./components/Planner.js";
 // import * as apiClient from "./apiClient";
 
 const App = () => {
-  const [isSignedIn, setIsSignedIn] = React.useState(false);
-  const [user, setUser] = React.useState({});
-
-  function logOut() {
-    setUser({});
-    setIsSignedIn(false);
-    console.log("logout successful");
-  }
+  const [isAuthenticated, setIsAuthenticated] = React.useState(gcal.sign);
 
   React.useEffect(() => {
-    console.log(user);
-  }, [user, isSignedIn]);
+    gcal.onLoad(() => {
+      setIsAuthenticated(gcal.gapi.auth2.getAuthInstance().isSignedIn.get());
+      gcal.listenSign((sign) => setIsAuthenticated(sign));
+    });
+  }, []);
 
   return (
     <main className="App">
       <h1>Here it is!</h1>
-      <Account
-        user={user}
-        setUser={(e) => setUser(e)}
-        logOut={() => logOut()}
-        isSignedIn={isSignedIn}
-        setIsSignedIn={(e) => setIsSignedIn(e)}
-      />
-      {isSignedIn ? <Planner /> : <p>Yo, sign in already.</p>}
+      <Login {...{ isAuthenticated }} />
```

Comment:
If your goal is to pass `isAuthenticated` as a prop to the `Login` component, I think you want `<Login isAuthenticated={isAuthenticated} />` here.  It's a more conventional way to pass props to a component and your teammates at your future companies will find it easier to read. :-)

EDIT- I see this specific code diff above is outdated, but the above method of adding props to components is still in-use, so I'll leave the comment in.